server/platform/services/sharedchannel/sync_send.go

```diff
@@ -53,13 +61,13 @@ func (scs *Service) NotifyChannelChanged(channelID string) {
 		return
 	}
 
-	task := newSyncTask(channelID, "", nil)
+	task := newSyncTask(channelID, "", "", nil, nil)
 	task.schedule = time.Now().Add(NotifyMinimumDelay)
 	scs.addTask(task)
 }
 
-// NotifyUserProfileChanged is called to indicate that a user belonging to at least one
-// shared channel has modified their user profile (name, username, email, custom status, profile image)
+// NotifyUserProfileChanged is called to indicate that a user has modified their user
```

Comment:
Updated because we call `onUserProfileChange` (and then, `NotifyUserProfileChanged`) without checking if the user belongs to a shared channel, filtering happens on `GetUsersForUser`